File: 557.反转字符串中的单词-iii.java
/*
 * @lc app=leetcode.cn id=557 lang=java
 *
 * [557] 反转字符串中的单词 III
 */

// @lc code=start
class Solution {
    public String reverseWords(String s) {
        String[] li = s.split(" ");
        for (int i = 0; i < li.length; i++) {
            StringBuilder b = new StringBuilder(li[i]);
            li[i] = b.reverse().toString();
        }
        StringBuilder sb = new StringBuilder(li[0]);
        for (int i = 1; i < li.length; i ++) {
            sb.append(" ");
            sb.append(li[i]);
        }
        return sb.toString();
    }
}
// @lc code=end

File: 557.反转字符串中的单词-iii.py
#
# @lc app=leetcode.cn id=557 lang=python3
#
# [557] 反转字符串中的单词 III
#

# @lc code=start


class Solution:
    def reverseWords(self, s: str) -> str:
        li = s.split(' ')
        for i in range(len(li)):
            li[i] = li[i][::-1]
        return ' '.join(li)

# @lc code=end
